### DISCUSSION TITLE
issue 2 hours and 16 minutes into the course

### DISCUSSION THREAD

**Discussion by dariocodes** [smartcontractkit/full-blockchain-solidity-course-py#1009] [Q&A] [answered]
When starting with lesson 2 I've run into an error I can't figure out using this code:

```Solidity
//SPDX-License-Identifier: MIT

pragma solidity >=0.6.0;

import "./simplestorage.sol";
``
contract StorageFactory {
    function createsimplestoragecontract() public {
        simplestorage _simplestorage = new simplestorage();
    }

}

**I'm running into this error:**
from solidity:
DeclarationError: Identifier not found or not unique.
 --> contracts/storageFactory.sol:9:9:
  |
9 |         simplestorage _simplestorage = new simplestorage();
  |         ^^^^^^^^^^^^^
```

**Comment by cromewar** [comment, marked as answer]
Hello @dariocodes 
Contract (class) name should start with upper case so the correct syntax is:

 `SimpleStorage simpleStorage = new SimpleStorage();`

also on your contract you must use the simple storage contract:

`contract StorageFactory is SimpleStorage {}`

Finally I see you have this: 


`import "./simplestorage.sol";`

As I said the contract names should be uppercase, so be sure to change it to:  `SimpleStorage`

And the contract declaration should be like this:

`contract SimpleStorage {}`
